data-serving/scripts/import-data/package.json

```diff
@@ -5,7 +5,8 @@
   "main": "dist/index.js",
   "scripts": {
     "build": "tsc",
-    "lint": "tsc --noEmit && eslint '*/**/*.{js,ts,tsx}' --quiet --fix"
+    "lint": "tsc --noEmit && eslint '*/**/*.{js,ts,tsx}' --quiet --fix",
+    "start": "mongo dist/index.js"
```

Comment:
Is this a way to preload the script in mongo shell, or typo?

Reply:
yeah this is what actually runs the mongo cli with the script -- slight fix in next CL to run tsc first